cpp/include/helpers/egl_thread_state.hpp

```diff
@@ -98,13 +101,22 @@ namespace rive_android
 		sk_sp<GrDirectContext> createGrContext();
 		sk_sp<SkSurface> createSkSurface();
 		static void* getProcAddress(const char*);
-		void clearSurface();
 		bool configHasAttribute(EGLConfig, EGLint, EGLint) const;
 
 		EGLBoolean makeCurrent(EGLSurface surface) const
 		{
 			return eglMakeCurrent(mDisplay, surface, surface, mContext);
 		}
+
+		sk_sp<GrDirectContext> getGrContext()
```

Comment:
We make a GL context from a `GrDirectContext`, but the field is called `mSkContext` since it's the Skia context. 
Yeah the naming here could be ironed out a bit.  😄